api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/FhirMedicationRequestDaoImpl.java

```diff
@@ -99,6 +106,24 @@ protected void setupSearchParams(Criteria criteria, SearchParameterMap theParams
 		excludeDiscontinueOrders(criteria);
 	}
 	
+	private Optional<Criterion> handleStatus(TokenAndListParam tokenAndListParam) {
+		return handleAndListParam(tokenAndListParam, token -> {
+			if (token.getValue() != null) {
+				try {
+					// currently only handles "ACTIVE"
+					if (Medication.MedicationStatus.ACTIVE.toString().equals(token.getValue().toUpperCase())) {
+						return Optional.of(FhirDaoUtils.createActiveOrderCriterion(""));
```

Comment:
Personally, I think I'd prefer having a variation of this that takes no arguments, even if it just does this in the background.

Reply:
Fair, will fix